xAPI.md

```diff
@@ -509,12 +509,56 @@ __Verb__: Defines the action being done by the Actor within the Activity within
 
 ## 8.0 Getting More Out of xAPI
```

Comment:
Extending xAPI and having a community of practice are very different things; how about we bump both up a top level, so we have 'Extending xAPI' and 'Profiles and Communities of Practice' but get rid of 'Getting More Out of xAPI' (or any replacement)?